pool/app/utils/json.ml

```diff
@@ -6,3 +6,12 @@ let read_variant_opt of_yojson m =
   try Some (read_variant of_yojson m) with
   | _ -> None
 ;;
+
+let[@warning "-8"] find_in_json_assoc_opt (json : Yojson.Safe.t) search_key =
```

Comment:
This could lead to a runtime error if the code below is refactored and doesn't include a catch-all.

Since it includes a catch all, we can remove the `[@warning "-8"]`.